### DISCUSSION TITLE
Citation key pinned in Zotero 7

### DISCUSSION THREAD

**Discussion by joergpilawa** [retorquere/zotero-better-bibtex#3047] [Q&A] [answered]
I just updated to Zotero 7.
Is there a way to see if a citation key is pinned or automatically generated? In the previous Zotero version there was this red pin. Is there an equivalent in Zotero 7?

**Comment by retorquere** [comment, marked as answer]
it's viewable by adding the citation key in the middle table, but I forgot to add it to the citation key section in the item detail display. A release build is running that will fix that.

**Reply by joergpilawa** [reply]
Thank you so much!
